Source/MoreDotNet.Test/Extensions/Numeric/RomanNumeralExtensions/ToRomanNumeralStringTests.cs

```diff
@@ -0,0 +1,311 @@
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given501_ShouldReturnDI()
+        {
+            var expected = "DI";
+            var input = 501;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given649_ShouldReturnDCXLIX()
+        {
+            var expected = "DCXLIX";
+            var input = 649;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given798_ShouldReturnDCCXCVIII()
+        {
+            var expected = "DCCXCVIII";
+            var input = 798;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given891_ShouldReturnDCCCXCI()
+        {
+            var expected = "DCCCXCI";
+            var input = 891;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given1000_ShouldReturnM()
+        {
+            var expected = "M";
+            var input = 1000;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given1004_ShouldReturnMIV()
+        {
+            var expected = "MIV";
+            var input = 1004;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given1006_ShouldReturnMVI()
+        {
+            var expected = "MVI";
+            var input = 1006;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given1023_ShouldReturnMXXIII()
+        {
+            var expected = "MXXIII";
+            var input = 1023;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given2014_ShouldReturnMMXIV()
+        {
+            var expected = "MMXIV";
+            var input = 2014;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given3999_ShouldReturnMMMCMXCIX()
+        {
+            var expected = "MMMCMXCIX";
+            var input = 3999;
+            var actual = input.ToRomanNumeralString();
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void ToRomanNumeralString_Given0_ShouldThrowArgumentOutOfRangeException()
```

Comment:
Same as the above, but here you can use the [InlineData] attribute because you have only 2 values for the exception cases.